### PR TITLE
curvebs/mds: add getChunkserverInCluster rpc api

### PR DESCRIPTION
**Pull request by SeanHai**
Signed-off-by: wanghai01 <seanhaizi@163.com>

### What problem does this PR solve?

Issue Number: #xxx 

Problem Summary:

### What is changed and how it works?

What's Changed:

How it Works:

Side effects(Breaking backward compatibility? Performance regression?):

### Check List

- [ ] Relevant documentation/comments is changed or added
- [ ] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

**Comment by SeanHai** [comment]
cicheck

**Comment by SeanHai** [comment]
cicheck

**Comment by SeanHai** [comment]
cicheck

**Comment by SeanHai** [comment]
cicheck

File: src/mds/topology/topology_service.cpp [hunk 1]
```diff
@@ -127,6 +127,39 @@ void TopologyServiceImpl::GetChunkServer(
...
+    google::protobuf::RpcController* cntl_base,
+    const GetChunkServerInClusterRequest* request,
+    GetChunkServerInClusterResponse* response,
+    google::protobuf::Closure* done) {
+    brpc::ClosureGuard done_guard(done);
+
+    brpc::Controller* cntl =
+        static_cast<brpc::Controller*>(cntl_base);
+
+    LOG(INFO) << "Received request[log_id=" << cntl->log_id()
```
Comment (wu-hanqing) [inline src/mds/topology/topology_service.cpp:140]: Will this function be called frequently? If so, these logs may be a bit too much.

File: src/mds/topology/topology_service.h [hunk 2]
```diff
@@ -59,6 +59,12 @@ class TopologyServiceImpl : public TopologyService {
                       GetChunkServerInfoResponse* response,
                       google::protobuf::Closure* done);
 
+    virtual void GetChunkServerInCluster(
+        google::protobuf::RpcController* cntl_base,
+        const GetChunkServerInClusterRequest* request,
+        GetChunkServerInClusterResponse* response,
+        google::protobuf::Closure* done);
```
Comment (wu-hanqing) [inline src/mds/topology/topology_service.h:66]:
```suggestion
   void GetChunkServerInCluster(
        google::protobuf::RpcController* cntl_base,
        const GetChunkServerInClusterRequest* request,
        GetChunkServerInClusterResponse* response,
        google::protobuf::Closure* done) override;
```

File: src/mds/topology/topology_service.cpp (hunk 1 above)
Comment (SeanHai) [reply src/mds/topology/topology_service.cpp]: fixed

**Comment by SeanHai** [comment]
cicheck

**Comment by SeanHai** [comment]
cicheck

**Review by wu-hanqing — APPROVED** [review]

**Review by Cyber-SiKu — APPROVED** [review]

**Pull request merged**